plugins/win-capture/game-capture.c

```diff
@@ -2022,6 +2006,15 @@ static void game_capture_render(void *data, gs_effect_t *effect)
 	effect = obs_get_base_effect(gc->config.allow_transparency
 					     ? OBS_EFFECT_DEFAULT
 					     : OBS_EFFECT_OPAQUE);
+	
+	if (gc->config.mode == CAPTURE_MODE_AUTO) {	
+		struct obs_video_info ovi;	
+		obs_get_video_info(&ovi);
+		float cx_scale = ovi.base_width/(float)gc->cx;
+		float cy_scale = ovi.base_height/(float)gc->cy;
+		gs_matrix_push();
+		gs_matrix_scale3f(cx_scale, cy_scale, 1.0f);
+	}
```

Comment:
2 questions:
1. Is it the action that fits the source to the screen?
2. Is it executed on each `game_capture_render`?

My concern is that these 2 are true, that would mean we are fitting to the screen all the time and we don't let the user rescale the source.

Reply:
gs_matrix_scale3f is add scale transformation not overwrite it. 
So it keeps user scale and add additional one to compensate changes in resolution between different games.

Reply:
Got it, makes sense